src/styles/header.scss

```diff
@@ -19,6 +20,7 @@
 
 	/* logo */
 	#header,
```

Comment:
is this correct? this would get converted into `#header #header` when converting from `scss` to `css`. Should probably be `.erc-header__logo` I think?